packages/issuer-rest/lib/IssuerTokenEndpoint.ts

```diff
@@ -37,16 +44,58 @@ export const handleTokenRequest = <T extends object>({
       })
     }
 
+    if (request.headers.authorization && request.headers.authorization.startsWith('DPoP ') && !request.headers.DPoP) {
+      return sendErrorResponse(response, 400, {
+        error: TokenErrorResponse.invalid_request,
+        error_description: 'DPoP header is required',
+      })
+    }
+
+    let dPoPJwk: JWK | undefined
+    if (requireDPoP && !request.headers.dpop) {
+      return sendErrorResponse(response, 400, {
+        error: TokenErrorResponse.invalid_request,
+        error_description: 'DPoP is required for requesting access tokens',
+      })
+    }
+
+    if (request.headers.dpop) {
+      if (!dPoPVerifyJwtCallback) {
+        return sendErrorResponse(response, 400, {
```

Comment:
Do we want to log something or make the error a bit more clear. There is a distinction in not enabling DPOP and not having provided the verify callback. In other words, for anyone configuring the issuer, this might be a confusing error

Reply:
I made the drop input params an object and logged an error for the issuer controller.